star-rx72n-firmware/lib/rx_hal/src/rx_mpc.c

```
@@ -302,43 +315,47 @@ rx_err_t rx_mpc_set_mtu_encoder(gpio_pin_t pin)
-  uint8_t pin_num = gpio_pin_get_pin(pin);
+  uint8_t port    = rx_port_from_pin(pin);
+  uint8_t pin_num = rx_pin_from_pin(pin);
 
   /* ADC mode: Set ASEL = 1 to disable digital I/O
    * Common ADC pins: P40-P47 (AN000-AN007)
    */
   return internal_write_pfs(port, pin_num, k_pfs_asel);
 }
 
-rx_err_t rx_mpc_set_sci(gpio_pin_t pin, bool is_tx)
+rx_err_t rx_mpc_set_sci(rx_port_pin_t pin, bool is_tx)
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'rx_mpc_set_sci' of similar type ('int') are easily swapped by mistake

(bugprone-easily-swappable-parameters)